lib/src/request/builder.rs

```diff
@@ -4,7 +4,9 @@
 use crate::chip::Chip;
 use crate::line::{self, Bias, Direction, Drive, EdgeDetection, EventClock, Offset, Value};
 use crate::request::{Config, Request};
-use crate::{AbiVersion, Error, Result, UapiCall};
+use crate::{Error, Result, UapiCall};
+#[cfg(feature = "uapi_v1")]
+use crate::AbiVersion;
```

Comment:
Ditto: ^^^ AbiVersion is only used if the "uapi_v1" feature is defined, either by itself or with both defined.